presto-main/src/main/java/com/facebook/presto/sql/analyzer/StatementAnalyzer.java

```diff
@@ -350,8 +351,12 @@ protected RelationType visitDescribeInput(DescribeInput node, AnalysisContext co
         // get all parameters in query
         List<Parameter> parameters = getParameters(statement);
 
-        // return the positions and types of all parameters
+        // return the positions and types of all parameters.  If there are no parameters, return a single null row.
         Row[] rows = parameters.stream().map(parameter -> createDescribeInputRow(parameter, queryAnalysis)).toArray(Row[]::new);
+        if (rows.length == 0) {
+            rows = new Row[] {row(new NullLiteral(), new NullLiteral())};
```

Comment:
I don't have a strong preference. This means I'll have to change my code to return a row of NULLs for DDL/DML statements. NULL it is then.